files/ko/learn/javascript/first_steps/useful_string_methods/index.md

```diff
@@ -55,110 +55,6 @@ var string = 'This is my string';
 
 콘솔에 예제를 작성해 보세요. 아래의 한 가지를 제공합니다(새 탭이나 새 창에서 콘솔을 열 수 있고, 브라우저의 개발자 콘솔을 사용할 수도 있습니다).
 
-<div class="hidden"><h6 id="Hidden_code">Hidden code</h6><pre class="brush: html notranslate">&#x3C;!DOCTYPE html>
```

Comment:
Looks like the conversion from HTML had a problem, so copied the current from the content repo